searchlib/src/vespa/searchlib/tensor/direct_tensor_store.cpp

```diff
@@ -31,8 +57,7 @@ EntryRef
 DirectTensorStore::store_tensor(std::unique_ptr<Tensor> tensor)
 {
     assert(tensor);
-    // TODO: Account for heap allocated memory
-    return _concrete_store.addEntry(TensorSP(tensor.release()));
+    return add_entry(TensorSP(tensor.release()));
```

Comment:
I would prefer making the shared_ptr as TensorSP(std::move(tensor)) instead of using release().

Reply:
Fixed